soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -1608,10 +1608,14 @@ void FileChoose_LoadGame(GameState* thisx) {
     u16 swordEquipMask;
     s32 pad;
 
-    if (this->buttonIndex == FS_BTN_SELECT_FILE_1 && CVar_GetS32("gDebugEnabled", 0)) {
+    if ((this->buttonIndex == FS_BTN_SELECT_FILE_1 && CVar_GetS32("gDebugEnabled", 0)) || this->buttonIndex == 0xFF) {
         Audio_PlaySoundGeneral(NA_SE_SY_FSEL_DECIDE_L, &D_801333D4, 4, &D_801333E0, &D_801333E0, &D_801333E8);
         gSaveContext.fileNum = this->buttonIndex;
-        Sram_OpenSave();
+        if (this->buttonIndex == 0xFF) {
+            Sram_InitDebugSave();
```

Comment:
it's not clear what's happening here, will this not get hit if we already created a debug save in file 1?

Reply:
A debug save is not tied to a file slot, this for the 4th option specifically. The underlying code is already set up to handle fileNum = 255 (0xFF)